game-core/src/main/java/games/strategy/io/IoUtils.java

```diff
@@ -60,61 +69,12 @@ public static void consumeFromMemory(final byte[] bytes, final InputStreamConsum
    *
    * @throws IOException If {@code consumer} encounters an error while writing to the output stream.
    */
-  public static byte[] writeToMemory(final OutputStreamConsumer consumer) throws IOException {
+  public static byte[] writeToMemory(final ThrowingConsumer<OutputStream, IOException> consumer) throws IOException {
```

Comment:
@ssoloff I think this change is causing compatibility issues and bots to freeze up: https://forums.triplea-game.org/topic/716/bot-63-stuck/26